src/locales/en_US.js

```diff
@@ -722,7 +722,9 @@ const strings = {
   fio_wallet_connect_return: 'Do not remove',
   fio_show_connected_wallets: 'Show connected',
   fio_connect_no_wallets: 'There are no wallets',
-  fio_connect_checkbox_text: 'I acknowledge that once connected, addresses from these wallets will become public',
+  fio_connect_only_5_per_once: 'You can connect up to 5 wallets per once',
```

Comment:
Same comment as above. Dislike the select wallet limitation. As well, this string is grammatically incorrect.